tests/screenshot/apps/components/Input.js

```diff
@@ -33,7 +34,18 @@ const InputTests = [
 	// Small input
 	<Input value="Simple value" size="small" />,
 	<Input value="Simple value" iconBefore="happyface" size="small" />,
-	<Input value="Simple value" iconAfter="happyface" size="small" />
+	<Input value="Simple value" iconAfter="happyface" size="small" />,
+
+	// Focus
+	...withConfig({focus: true}, [
+		<Input placeholder="Focused placeholder Input" />,
+		<Input placeholder="Focused placeholder Input" disabled />,
+		<Input value="Focused simple value" />,
+		<Input value="Focused simple value" disabled />,
+		<Input invalid invalidMessage="Focused custom invalid message" />,
```

Comment:
I don't know why but.. this test is not showing the invalid tooltip. Maybe it's rendered out of the screen...?!

Reply:
I changed the tests to show the tooltip also.